assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -510,25 +517,27 @@ function BannerNotification( {
 						) }
 					</Cell>
 
-					{ WinImageSVG && (
-						<Cell
-							{ ...imageCellSizeProperties }
-							{ ...imageCellOrderProperties }
-							alignBottom={
-								format === 'larger' && noBottomPadding
-							}
-							className="googlesitekit-display-none googlesitekit-non-mobile-display-block"
-						>
-							<div
-								className={ `googlesitekit-publisher-win__image-${ format }` }
+					{ WinImageSVG &&
+						( isMinWidthTablet || ! showSmallWinImage ) && (
```

Comment:
Avoiding rendering the SVG twice in the page. See **Relevant technical choices** in the PR description / https://github.com/google/site-kit-wp/commit/22d62c79a003ce7a6ef7be447c11e9497dfb87b5.